packages/thirdweb/src/contract/actions/compiler-metadata.ts

```diff
@@ -7,7 +7,9 @@ export type PublishedMetadata = {
   name: string;
   abi: Abi;
   // biome-ignore lint/suspicious/noExplicitAny: TODO: fix later
-  metadata: Record<string, any>;
+  metadata: Record<string, any> & {
```

Comment:
I believe we should type this based on [the spec](https://docs.soliditylang.org/en/latest/metadata.html) ?

Reply:
@joaquim-verges what do u think? I think we should be able to rely on this right?

Reply:
yes we should

Reply:
wouldnt block on it though - can you leave a TODO and link to the spec @kien-ngo

Reply:
Yep got it